Project.toml

```
@@ -12,7 +12,7 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
 DataFrames = "0.21"
-FITSIO = "0.14.0, 0.15"
+FITSIO = "0.14.0, 0.15, 0.16"
```

@mileslucas maybe let's change to
```suggestion
FITSIO = "0.14.0, 0.15, 0.16.1"
```
after https://github.com/JuliaAstro/FITSIO.jl/pull/139 is merged and registered?

---

I agree, we should avoid using 0.16.0. We should resolve this alongside #31, though, since it uses some of the deprecated API currently. cc @siddharthlal25

---

This would make slightly less probable that someone installs FITSIO.jl v0.16.0, but in the remote case someone has an environment with FITSIO.jl v0.16.0, installing CCDReduction would be a bit tricky without an update.  Hopefully, no one will ever install the "bugged" version of FITSIO.jl